apps/system/js/hierarchy_manager.js

```diff
@@ -58,6 +58,7 @@
      * @type {Array}
      */
     PRIORITIES: [
+      'OverlayWindowManager',
```

Comment:
Okay you have this. You need to tell HM when you are active and when you are inactive by publish activated/deactivated. However I am still not sure what's the desired behavior of GOWM - does it affect screen reader? does it need to hide the bottom windows for screen reader?